easybuild/easyblocks/c/cuda.py

```diff
@@ -234,6 +234,17 @@ def create_wrapper(wrapper_name, wrapper_comp):
             raise EasyBuildError("Unable to find 'ldconfig' in $PATH (%s), nor in any of %s", path, sbin_dirs)
 
         stubs_dir = os.path.join(self.installdir, 'lib64', 'stubs')
+
+        # Remove stubs which are not required as the full library is in $EBROOTCUDA/lib64 because this duplication
+        # causes issues (e.g. CMake warnings) when using this module (see $LIBRARY_PATH & $LD_LIBRARY_PATH)
+        for stub_lib in expand_glob_paths([os.path.join(stubs_dir, '*.*')]):
```

Comment:
Are both removals required to get rid of the warnings?

Reply:
After looking at https://github.com/easybuilders/easybuild-easyblocks/pull/2373/files, I'm guessing not since we only add our copy to the `LIBRARY_PATH`

Reply:
Yes, hence the separate commit. I thought the same but when I opened the PR I noticed that CMake complains about `lib/stubs` not `stubs/lib` (the latter is ours) so without this change we wouldn't fix the CMake warning.

The warning is actually caused by PyTorch linking against `libcuda` which is searched for in the nvcc installation prefix first which finds the libcuda in `lib/stubs`. So creating a single search path would not only bring in the (stub) libcuda but also all the other stub libs. This isn't ideal at all and I'm likely filing a bug report for that.

Anyway: I can't imagine how removing the stubs could lead to any issues because we have the full libs and only remove the stubs when there is a full lib. So I'd say its fine

Reply:
Ok, I'm in general agreement and haven't heard any grumbles from other maintainers.